### PR TITLE
Fix documentation task on new repositories

### PR DESCRIPTION
**Pull request by bquorning**
**Replace this text with a summary of the changes in your PR.
The more detailed you are, the better.**

-----------------

Before submitting the PR make sure the following are checked:

* [ ] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences.
* [ ] Wrote [good commit messages][1].
* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [ ] Feature branch is up-to-date with `master` (if not - rebase it).
* [ ] Squashed related commits together.
* [ ] Added tests.
* [ ] Ran `bundle exec rake default`. It executes all tests and runs RuboCop on its own code.
* [ ] Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.

[1]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by bbatsov** [comment]
@bquorning Any plans to work on this in the future?

**Comment by bquorning** [comment]
No, probably not.

File: lib/rubocop/cops_documentation_generator.rb [hunk 1]
```diff
@@ -298,7 +302,10 @@ def table_of_content_for_department(department)
   end
 
   def print_table_of_contents
-    path = "#{Dir.pwd}/docs/modules/ROOT/pages/cops.adoc"
+    path = "#{docs_path}/cops.adoc"
+
+    File.write(path, table_contents) and return unless File.exist?(path)
```
Comment (bquorning) [inline lib/rubocop/cops_documentation_generator.rb:307]: I personally prefer `if !File.exist?(path)` but 🤷🏼

**Review by bbatsov — APPROVED** [review]

**Comment by bbatsov** [comment]
Looks good. Thanks!

**Pull request closed without merging**